### ISSUE TITLE
[BUG] Extraction -> quarantine application lasts minutes

### ISSUE THREAD

**Issue by xsvlz** [aonez/Keka#701]
### Configuration
- Keka version:  1.2.3 (4223)
- macOS version:10.15.7 (19H15)

### Describe the bug
When extract files using Keka, it always stay at this step:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/52330309/100365972-2c567780-303b-11eb-9141-b1adefc68398.png">

extract any file will spend several minutes, but it can be extracted immediately by other tools.

keka log:
```
Format detected: ZIP (7Z)
Binary used: keka7z
Arguments: (
    x,
    "/Users/xsv/Downloads/PicGo-2.3.0-beta.3-mac.zip",
    "-o/Users/xsv/Downloads/PicGo-2.3.0-beta.3-mac.kextraction/Operation",
    "-aou",
    "-xr!__MACOSX"
)

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=utf8,Utf16=on,HugeFiles=on,64 bits,8 CPUs x64)
Modified by aONe for Keka

Scanning the drive for archives:
  0M Scan /Users/xsv/Downloads/

1 file, 65475450 bytes (63 MiB)

Extracting archive: /Users/xsv/Downloads/PicGo-2.3.0-beta.3-mac.zip
 91% 256 Open


WARNINGS:
There are data after the end of archive

--
Path = /Users/xsv/Downloads/PicGo-2.3.0-beta.3-mac.zip
Type = zip
WARNINGS:
There are data after the end of archive
Physical Size = 65405345
Tail Size = 70105

 33%

Everything is Ok

Archives with Warnings: 1

Warnings: 1
Folders: 146
Files: 186
Size:       175811814
Compressed: 65475450

```

**Comment by aonez** [comment]
@xsvlz can you update to the latest 1.2.4 version? Just released it today. If it still hangs there can you check on the Console.app if you see some messages [like this](https://github.com/aonez/Keka/issues/176#issuecomment-728148463):

```
default	16:50:09.540168 +0100	Keka	database is locked
```

As a workaround please disable the quarantine application in the Keka preferences:

![Screenshot 2020-11-26 at 16 10 26](https://user-images.githubusercontent.com/480163/100367056-f47f0e00-3001-11eb-8bab-a342869425a5.png)

**Comment by aonez** [comment]
This was first reported in https://github.com/aonez/Keka/issues/176#issuecomment-726653298 and in theory fixed in 1.2.3... Sad to see this happen for you in that version :(
